### PR TITLE
use Wear Compose Rotary APIs

### PR DESCRIPTION
**Pull request by yschimke**
#### WHAT

User Wear Compose Rotary APIs.

#### WHY

Use standard implementations, and don't hide behind Horologist APIs.

#### Checklist :clipboard:
- [ ] Add explicit visibility modifier and explicit return types for public declarations
- [ ] Run spotless check
- [ ] Run tests
- [ ] Update metalava's signature text files

### CODE REVIEW

File: composables/src/main/java/com/google/android/horologist/composables/TimePicker.kt [hunk 1]
```diff
@@ -297,7 +295,7 @@ public fun TimePicker(
                     *pickerGroupItems.toTypedArray(),
                     modifier = Modifier.fillMaxWidth(),
                     pickerGroupState = pickerGroupState,
-                    expandToFillWidth = showSeconds,
+//                    expandToFillWidth = showSeconds,
```
Comment (ssancho14) [inline composables/src/main/java/com/google/android/horologist/composables/TimePicker.kt:298]: Remove?

**Review by ssancho14 — COMMENTED** [review]
Looks good, defering approval to Misha that knows the rotary code better.

File: media/audio-ui/src/main/java/com/google/android/horologist/audio/ui/RotaryVolumeControls.kt [hunk 2]
```diff
@@ -117,6 +132,22 @@ public fun Modifier.lowResRotaryVolumeControls(
  * accumulated scrolled pixels to volume to pass into [onRotaryVolumeInput] for a corresponding
  * volume change
  */
+@Deprecated(
+    "Replaced by wear compose",
```
Comment (Kpeved) [inline media/audio-ui/src/main/java/com/google/android/horologist/audio/ui/RotaryVolumeControls.kt:136]: Isn't the official name of our library is "Compose for Wear OS" ? :)

File: composables/src/main/java/com/google/android/horologist/composables/TimePicker.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (yschimke) [reply composables/src/main/java/com/google/android/horologist/composables/TimePicker.kt]: It's not in the Wear Compose API.

File: media/audio-ui/src/main/java/com/google/android/horologist/audio/ui/RotaryVolumeControls.kt (hunk 2 above)
Comment (yschimke) [reply media/audio-ui/src/main/java/com/google/android/horologist/audio/ui/RotaryVolumeControls.kt:136]: I should clean this up, ooops.

**Pull request merged**